reference/format/dashboard.rst

```diff
@@ -0,0 +1,101 @@
+Dashboard
+=========
+
++-----------+--------------------------+
+| Filename  | ``dashboard.yml``        |
```

Comment:
Each dashboard widget can have configuration fields mapping with `configuration` parameter. Please add this parameter description to this file. Some info about this parameter you can find here https://github.com/laboro/platform/tree/master/src/Oro/Bundle/DashboardBundle#widget-configuration